Source/JavaScriptCore/runtime/TemporalCalendarPrototype.cpp

```diff
@@ -401,7 +401,7 @@ JSC_DEFINE_HOST_FUNCTION(temporalCalendarPrototypeFuncToJSON, (JSGlobalObject* g
     if (!calendar)
         return throwVMTypeError(globalObject, scope, "Temporal.Calendar.prototype.toJSON called on value that's not a Calendar"_s);
 
-    return JSValue::encode(calendar->toString(globalObject));
+    RELEASE_AND_RETURN(scope, calendar->toString(globalObject));
```

Comment:
Er whoops, we still need the `JSValue::encode()` in there.

Reply:
Oops, fixed.